Educational_project/Services/OrderService.cs

```diff
@@ -1,16 +1,24 @@
-﻿using StorePhone.Models;
+﻿using EF_Store.Data.Contracts;
+using StorePhone.Models;
 using StorePhone.Сontracts;
 using System;
 using System.Linq;
 
 namespace StorePhone.Controllers
 {
-    public class OrderController : IOrderController
+    public class OrderService : IOrderService
     {  
         private readonly IDbContext _dbContext;
         private readonly ILogger _logger;
 
-        public OrderController(IDbContext dbContext, ILogger logger)
+        private readonly IUnitOfWork _unitOfWork;
+
+        public OrderService(IUnitOfWork unitOfWork)
```

Comment:
use service from BLL here
use IUnitOfWork  inside the service in BLL
remove IUnitOfWork from controller